compiler/rustc_typeck/src/outlives/mod.rs

```diff
@@ -59,12 +61,7 @@ fn inferred_outlives_of(tcx: TyCtxt<'_>, item_def_id: DefId) -> &[(ty::Predicate
                         .collect();
                     pred.sort();
 
-                    let span = tcx.def_span(item_def_id);
-                    let mut err = tcx.sess.struct_span_err(span, "rustc_outlives");
```

Comment:
pre-existing: but this is a bizarre error message.. and the blame is from many years ago.